src/fastoad/models/propulsion/fuel_propulsion/rubber_engine/openmdao.py

```diff
@@ -94,6 +94,16 @@ def setup(self, component: Component):
             desc="As it is a delta, unit is K or °C, but is not "
             "specified to avoid OpenMDAO making unwanted conversion",
         )
+        component.add_input(
+            "tuning:propulsion:rubber_engine:SFC:k_sl",
+            1.0,
+            desc="correction ratio to apply to the computed SFC at sea level",
```

Comment:
Here also, now that the descriptions are in variable_descriptions.txt, we should stick to the [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principle and remove the `desc` fields here and below.

Reply:
The modification was made